docs/cqlengine/queryset.rst

```diff
@@ -199,6 +199,14 @@ In addition to simple equal to queries, cqlengine also supports querying with ot
 
         q = Automobile.objects.filter(model__like='%Civic%').allow_filtering()
 
+:attr:`IS NOT NULL (IsNotNull(column_name)) <query.QueryOperator.IsNotNullOperator>`
```

Comment:
This should probably point to the Clause and not the Operator. In this case, I think the Operator can't be used directly.

Reply:
That's correct, the clause has to be used for this case. I'll change it